app/post.go

```diff
@@ -17,13 +20,35 @@ import (
 )
 
 var (
-	c = &http.Client{
-		Timeout: 5 * time.Second,
-	}
+	httpClient *http.Client
 
+	httpTimeout       = time.Duration(10 * time.Second)
```

Comment:
China's bandwidth to oversea networks is very limited, I tried to adjust different timeout to fix our needs, if you feel it unnecessary I will reset it back.